### PR TITLE
Spectrum plot tweaks

### PR DESCRIPTION
**Pull request by guynir42**
- Fixed MANAGE DATA button that was hiding the bottom of Bokeh plot.
- Fixed redshift and expansion velocity inputs to accept any accuracy inputs using numeric input widgets.
Screenshot from 2022-01-17 15-56-23

### CODE REVIEW

**Comment by guynir42** [comment]
addresses #2563 and #2386

File: skyportal/plot.py [hunk 1]
```diff
@@ -1723,27 +1723,31 @@ def make_spectrum_layout(obj, spectra, user, device, width, smoothing, smooth_nu
...
+        step=0.00001,
         show_value=False,
-        format="0[.]000",
+        format="0[.]0000",
     )
-    z_textinput = TextInput(
-        value=str(obj.redshift if obj.redshift is not None else 0.0)
+    # z_textinput = TextInput(
+    #     value=str(obj.redshift if obj.redshift is not None else 0.0)
+    # )
```
Comment (profjsb) [inline skyportal/plot.py:1732]:
```suggestion

```

**Review by profjsb — CHANGES_REQUESTED** [review]
does this entirely close #2563 and #2386?

**Comment by guynir42** [comment]
> does this entirely close #2563 and #2386?

@profjsb I believe so.

**Pull request merged**